File: android/app/src/main/java/com/enterslice/enterslice/DatabaseHelper.java
package com.enterslice.enterslice;



import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import androidx.annotation.Nullable;

public class DatabaseHelper extends SQLiteOpenHelper {
    public static final int DATABASE_VERSION = 1;
    public static final String DATABASE_NAME = "myDb";

    public DatabaseHelper(@Nullable Context context, @Nullable String name, @Nullable SQLiteDatabase.CursorFactory factory, int version) {
        super(context, name, factory, version);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL("create table location(latitude text, longitude text,time text)");
    }

    @Override
    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {

    }


}

File: lib/databaseHelper.dart
import 'package:path/path.dart';
import 'package:sqflite/sqflite.dart';

class DatabaseHelper{
  static late Database db;

  static openDb() async {
    var databasesPath = await getDatabasesPath();
    String path = join(databasesPath, 'myDb');

    db=await openDatabase(path,version: 1,onCreate: (db, version) async {
      await db.execute("create table location(latitude text, longitude text,time text)");
    },);
  }

  static closeDb() async {
    await db.close();
  }
}